### PR TITLE
spelling and grammatical changes

### PR DESCRIPTION
**Pull request by andrewbanchich**
fixes spelling and grammatical issues

### CODE REVIEW

File: docs/reference/docs/reindex.asciidoc [hunk 1]
```diff
@@ -161,11 +161,13 @@ POST _reindex
...
-`post` types in the `twitter` and `blog` index. It'd include the `post` type in
-the `twitter` index and the `_doc` type in the `blog` index. If you want to be
-more specific you'll need to use the `query`. It also makes no effort to handle
-ID collisions. The target index will remain valid but it's not easy to predict
-which document will survive because the iteration order isn't well defined.
+`post` types in the `twitter` and `blog` index. The copied documents would include the 
+`post` type in the `twitter` index and the `_doc` type in the `blog` index. For more 
+specific parameters, you can use `query`.
+
+The Reindex API makes no effort handle ID collisions. For such issues, the target index
```
Comment (cbuescher) [inline docs/reference/docs/reindex.asciidoc]: nit: "to handle"?

File: docs/reference/docs/reindex.asciidoc [hunk 2]
```diff
@@ -203,10 +205,10 @@ POST _reindex
 // CONSOLE
 // TEST[setup:twitter]
 
-If you want a particular set of documents from the twitter index you'll
-need to sort. Sorting makes the scroll less efficient but in some contexts
+If you want a particular set of documents from the `twitter` index you'll
+need to `sort`. Sorting makes the scroll less efficient but in some contexts
```
Comment (cbuescher) [inline docs/reference/docs/reindex.asciidoc]: nit: If this is not refering to the verb "sort" but to the dsl command in the following example, I would use "you'll need to use `sort`".

File: docs/reference/docs/reindex.asciidoc [hunk 3]
```diff
@@ -203,10 +205,10 @@ POST _reindex
...
 it's worth it. If possible, prefer a more selective query to `size` and `sort`.
-This will copy 10000 documents from `twitter` into `new_twitter`:
+This will copy 10,000 documents from `twitter` into `new_twitter`:
```
Comment (cbuescher) [inline docs/reference/docs/reindex.asciidoc]: nit: I'd leave this as is, the comma is used in many countries (e.g. in Germany and most of the EU) as the decimal separator, so to me personally this reads more like 10 (with a weird extra zero). The existing version might not be as good to read but is less ambiguous I think.

File: docs/reference/docs/reindex.asciidoc [hunk 4]
```diff
@@ -946,7 +946,7 @@ POST _reindex?slices=5&refresh
 // CONSOLE
 // TEST[setup:big_twitter]
 
-Which you also can verify works with:
+You can also verify works with:
```
Comment (cbuescher) [inline docs/reference/docs/reindex.asciidoc]: "it works" or "this works" maybe?

File: docs/reference/docs/reindex.asciidoc [hunk 5]
```diff
@@ -136,7 +136,7 @@ POST _reindex
 // TEST[setup:twitter]
 
 You can limit the documents by adding a type to the `source` or by adding a
-query. This will only copy ++tweet++&apos;s made by `kimchy` into `new_twitter`:
+query. This will only copy ++tweet++s made by `kimchy` into `new_twitter`:
```
Comment (cbuescher) [inline docs/reference/docs/reindex.asciidoc]: I wonder if you could close #29231 and use this PR for those changes instead.

**Review by cbuescher — CHANGES_REQUESTED** [review]
@andrewbanchich these are really a couple of great changes. I left some small suggestions and marked a few spots where I felt there might be something missing. Looking forward to the update.

File: docs/reference/docs/reindex.asciidoc (hunk 5 above)
Comment (andrewbanchich) [reply docs/reference/docs/reindex.asciidoc]: All set

File: docs/reference/docs/reindex.asciidoc (hunk 1 above)
Comment (andrewbanchich) [reply docs/reference/docs/reindex.asciidoc]: Good catch!

File: docs/reference/docs/reindex.asciidoc (hunk 2 above)
Comment (andrewbanchich) [reply docs/reference/docs/reindex.asciidoc]: Agreed

File: docs/reference/docs/reindex.asciidoc (hunk 3 above)
Comment (andrewbanchich) [reply docs/reference/docs/reindex.asciidoc]: Agreed

File: docs/reference/docs/reindex.asciidoc (hunk 4 above)
Comment (andrewbanchich) [reply docs/reference/docs/reindex.asciidoc]: Changed

**Comment by andrewbanchich** [comment]
@cbuescher All set with these changes!

File: docs/reference/docs/reindex.asciidoc (hunk 5 above)
Comment (andrewbanchich) [reply docs/reference/docs/reindex.asciidoc]: Forgot about that. Fixed!

**Review by cbuescher — APPROVED** [review]
@andrewbanchich that for the update, much appreciated. I will kick of a final test run before I merge.

**Comment by cbuescher** [comment]
@elasticmachine test this please

**Pull request merged**